src/containers/sidebars/featured-map-card/featured-map-card-styles.module.scss

```diff
@@ -12,13 +12,21 @@ $section-padding: 0 24px;
   flex-direction: column;
   width: $sidebar-width;
   height: auto;
-  border-radius: 2px;
+  border-radius: 12px;
   background-color: $brand-color-secondary;
+  overflow: hidden;
   @include animationFunction();
 
   @media #{$tablet-portrait} {
     display: flex;
   }
+
+  &.bestPlaces{
+    box-shadow: 0px 1px 8px #6FBE46;
```

Comment:
I would recommend using variables for these colors and others, like box-shadows, ...